packages/babel-polyfill/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "babel-polyfill",
   "version": "6.9.1",
-  "description": "",
+  "description": "provides polyfills necessary for a a full ES2015 environment",
```

Comment:
Can you change this to not use ES2015, because the lastest version is already ES2016 and it will change every year ;)
Not sure what to use though... maybe `ES2015+ environment`. Suggestions?

And please start the sentence with upper case to be in line with the other packages and there is `a a`